doc/guides/coding_guidelines/index.rst

```diff
@@ -823,3 +823,96 @@ Excluding declarations from the header based on compile-time options may prevent
 their documentation from being generated. Their absence also prevents use of
 ``if (IS_ENABLED(CONFIG_FOO)) {}`` as an alternative to preprocessor
 conditionals when the code path should change based on the selected options.
+
+Rule A.2: Inclusive Language
+============================
+
+Severity
+--------
+
+Required
+
+Description
+-----------
+
+Do not introduce new usage of offensive terms listed below. This rule applies
+but is not limited to source code, comments, documentation, and branch names.
+Replacement terms may vary by area or subsystem, but should aim to follow
+updated industry standards when possible.
+
+Exceptions are allowed for maintaining existing implementations of industry
+standard specifications governed externally to the Zephyr Project.
+
+Existing usage is recommended to change as soon as updated industry standard
+specifications become available, or immediately if no specifications apply.
```

Comment:
If the relevant authority has announced replacement terms do we have to wait for the updated specification to be published?

Reply:
Let's discuss this in next week's meeting